src/compat/stdin.cpp

```diff
@@ -3,18 +3,18 @@
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
 #if defined(HAVE_CONFIG_H)
-#include <config/bitcoin-config.h>
+#include <config/bitcoin-config.h> // IWYU pragma: keep
```

Comment:
why is this needed?

Reply:
> why is this needed? (I mean the include)

In this case, the include isn't actually needed. Although in other cases we'll likely use the same pragma on this same include to suppress IWYU output, given we aren't building for all configurations (I think that's ok). I'm removing it now.